mllib/src/test/scala/org/apache/spark/mllib/stat/HypothesisTestSuite.scala

```diff
@@ -254,4 +254,115 @@ class HypothesisTestSuite extends SparkFunSuite with MLlibTestSparkContext {
     assert(rCompResult.statistic ~== rKSStat relTol 1e-4)
     assert(rCompResult.pValue ~== rKSPVal relTol 1e-4)
   }
+
```

Comment:
If making the methods package private would simplify the test, I think it's reasonable to do so